content/docs/1.1.0/volumes-and-nodes/maintenance.md

```diff
@@ -24,8 +24,7 @@ If shutting down the workloads is not possible, follow the steps below to minimi
 
     The replica processes on the node will be stopped at this stage. 
         
-    > **Upcoming feature:** After adding the support of `Replica eviction`, you will be able to evict the replicas on the node gracefully.
-    
+    Use Disks or nodes eviction support to evict all the replicas on the node.
```

Comment:
This sequence doesn't make sense. The user should evict the replicas first, otherwise, they will be marked in error due to the Kubernetes drain process killing the replica instance manager pod.

Also, mention disabling the scheduling before eviction.